resources/extruders/annex_extruder_0.def.json

```
@@ -0,0 +1,16 @@
+{
+    "version": 1,

```

```suggestion
    "version": 2,
```
It won't work otherwise.